electrochemistry.ttl

```diff
@@ -286,6 +248,8 @@ emmo:EMMO_fd689787_31b0_41cf_bf03_0d69af76469d rdf:type owl:ObjectProperty .
                                                                          owl:someValuesFrom :electrochemistry_cf596ea1_c313_4fe1_bd7f_c18afc8a24e3
                                                                        ] ;
                                                        owl:disjointWith :electrochemistry_a9873d3d_945b_40ba_b9cd_8dfb64cec88f ;
+                                                       emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "operation during which an electrochemical cell supplies electric energy as the result of chemical changes within the cell"@en ;
```

Comment:
You would do it with a hasParticipant relation....hasAgent might also be applicable since the GalvanicCell is the main driver of the DischargeProcess. Probably best to keep it simple and say:

DischargeProcess hasParticipant some GalvanicCell 

hasConstitutiveProcess is used to link a Process to its SubProcess(es).